components/elective/EnrollmentIndicator.tsx

```diff
@@ -0,0 +1,49 @@
+import cn from "@/utils/helpers/cn";
+import { StylableFC } from "@/utils/types/common";
+import { Progress, Text } from "@suankularb-components/react";
+
+/**
+ * An indicator showing the enrollment status of an Elective Subject.
+ *
+ * @param classSize The number of students enrolled in the Elective Subject.
+ * @param capSize The maximum number of students that can be enrolled in the Elective Subject.
+ */
+const EnrollmentIndicator: StylableFC<{
+  classSize: number;
+  capSize: number;
+}> = ({ classSize, capSize, style, className }) => {
+  const isFull = classSize >= capSize;
+
+  return (
+    <div className={cn(`space-y-1`, className)} style={style}>
+      <Text
+        type="body-small"
+        className="block text-center text-on-surface-variant"
+      >
+        <Text
+          type="body-medium"
+          className={cn(`!font-medium`, isFull ? `text-error` : `text-primary`)}
+        >
+          {classSize}
+        </Text>
+        /{capSize}
+      </Text>
+
+      <Progress
+        appearance="linear"
+        alt="Enrollment"
```

Comment:
I'm not sure do you translate "alt", but if this is it, it's probably here where you forgot to translate this.

Reply:
Yep! Good catch. I’ll fix it and have you re-review.

Reply:
Please review this again.